hibernate-reactive-core/src/main/java/org/hibernate/reactive/mutiny/Mutiny.java

```diff
@@ -626,4 +632,37 @@
 		 */
 		void close();
 	}
+
+	/**
+	 * Asynchronously fetch an association that's configured for lazy loading.
+	 *
+	 * <pre>
+	 * {@code Mutiny.fetch(author.getBook()).thenAccept(book -> print(book.getTitle()));}
+	 * </pre>
+	 *
+	 * @param association a lazy-loaded association
+	 *
+	 * @return the fetched association, via a {@code CompletionStage}
+	 *
+	 * @see org.hibernate.Hibernate#initialize(Object)
+	 */
+	static <T> Uni<T> fetch(T association) {
```

Comment:
What if the proxy was disconnected from the Session ?

Reply:
Well it's actually not totally kosher to reassociate a proxy from one session with a different session Sanne. (JPA semantics vs legacy Hibernate semantics.)

Reply:
Sure - I'd never suggest anyone to do it :)  Just pointing out that it could happen.